models/category.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {object} sequelize
+ * @param {object} DataTypes
+ * @returns {object} Category
+ */
+const category = (sequelize, DataTypes) => {
+  const Category = sequelize.define('Category', {
+    title: DataTypes.STRING,
+    validate: {
+      isEmpty: {
+        msg: 'Empty title. The title is required.'
```

Comment:
Please can you check the custom error message for "isEmpty". You used firstname instead of title.